### PR TITLE
Changes to collect DNX statistics

### PR DESCRIPTION
**Pull request by shirhatti**
@xavierdecoster Hey Xavier, I made a couple of changes to allow for gathering DNX statistics.

I am still working on generating download count reports but I was hoping to get some initial feedback.

### CODE REVIEW

File: src/Stats.ImportAzureCdnStatistics/Dimensions/DnxDimension.cs [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+        }
+
+        public int Id { get; set; }
+        public string DnxVersion { get; }
+        public string OperatingSystem { get; }
+        public string FileName { get; }
+
+        protected bool Equals(DnxDimension other)
+        {
+            return string.Equals(DnxVersion, other.DnxVersion) && string.Equals(OperatingSystem, other.OperatingSystem) && string.Equals(FileName, other.FileName);
```
Comment (xavierdecoster) [inline src/Stats.ImportAzureCdnStatistics/Dimensions/DnxDimension.cs]: How about case-sensitivity here?

Comment (shirhatti) [inline src/Stats.ImportAzureCdnStatistics/Dimensions/DnxDimension.cs]: Case-sensitivity should not matter. I've updated the `Equals` function to reflect that

**Comment by xavierdecoster** [comment]
Changes look good to me. Can you please rebase them onto dev and squash your commits?

**Comment by shirhatti** [comment]
@glennc Could you have a look at this?

**Pull request closed without merging**